### ISSUE TITLE
Location of .gitbook

### ISSUE THREAD

**Issue by vstriz** [GitbookIO/gitbook-cli#20]
It would be nice to be able to override (with option or env variable) location of ".gitbook" dirctory and all versions being installed.  I need to put somewhere other than under my home directory.

**Comment by LionelCons** [comment]
+1

**Comment by obogobo** [comment]
Yes, us too. Our company uses shared machines for Jenkins slaves - having everyone's .gitbook folder in the same place is getting a little dicy
